addon/components/nypr-card/button.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import layout from '../../templates/components/nypr-card/button';
+
+const NyprCardButton = Ember.Component.extend({
+  layout,
+  tagName: 'button',
+  classNames: ['nypr-card-button'],
+  'data-test-selector': 'nypr-card-button'
+});
+
+NyprCardButton.reopen({
+  attributeBindings: ['data-test-selector']
+});
```

Comment:
oddly enough this is the method suggested by the docs: https://guides.emberjs.com/v2.12.0/templates/binding-element-attributes/#toc_adding-data-attributes